lib/active_record/json_validator/validator.rb

```diff
@@ -68,4 +69,8 @@ def fetch_schema_for_record(record)
 
     record.instance_exec(&schema)
   end
+
+  def validator_options
+    options.fetch(:validator_options)
+  end
```

Comment:
I think we can ditch this method and use `options.fetch(:validator_options)` straight like we use other options.